swatch-billable-usage/src/main/java/com/redhat/swatch/billable/usage/services/BillableUsageService.java

```diff
@@ -243,4 +252,31 @@ private void createRemittance(
         contractCoverage.isGratis() ? BillableUsage.Status.GRATIS : BillableUsage.Status.PENDING);
     usage.setUuid(newRemittance.getUuid());
   }
+
+  private void updateUsageMeter(BillableUsage usage, double contractCoverage, double billable) {
+    if (usage.getProductId() == null
+        || usage.getMetricId() == null
+        || usage.getBillingProvider() == null
+        || usage.getStatus() == null) {
+      return;
+    }
+    List<String> tags =
+        new ArrayList<>(
+            List.of(
+                "product_tag", usage.getProductId(),
```

Comment:
According to the google document linked to the epic, the tag should be "product".